packages/botonic-plugin-flow-builder/src/flow-builder-models.ts

```diff
@@ -126,6 +126,7 @@ export interface HandoffNodeContent {
   message: TextLocale[]
   fail_message: TextLocale[]
```

Comment:
this no longer exists on the handoff content. We should remove it in a future PR

Reply:
ok